negotiation/src/test/java/it/eng/negotiation/service/ContractNegotiationAPIServiceTest.java

```diff
@@ -61,7 +59,7 @@ public class ContractNegotiationAPIServiceTest {
     private CredentialUtils credentialUtils;
 	
 	@Captor
-	private ArgumentCaptor<ContractNegotiation> argumentCaptor;
+	private ArgumentCaptor<ContractNegotiation> argCaptorContractNegotiation;
```

Comment:
I was lazy to name it full :D

Reply:
I wouldn't have changed it either, but since we have two of them it caused me confusion :D